### DISCUSSION TITLE
Example of dendritic crystal growth

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#20882] [Q&A Modules: Phase field] [answered]
Hi, everyone.

I want to reproduce snowflake growth using 2 order parameters，like op = eta1, op = eta2,
the kenerl is ACInterfaceKobayashi1, ACInterfaceKobayashi2 and InterfaceOrientationMaterial.

But it  doesn't work：

*** ERROR ***
The following material properties are declared on block 0 by multiple materials:
  Material Property             Material Objects
  ddepsdgrad_op                 material material_2
  deps                          material material_2
  depsdgrad_op                  material material_2
  eps                           material material_2 

Hope to get your help.
Kang C R

**Comment by laagesen** [comment, marked as answer]
Please see https://github.com/idaholab/moose/discussions/20886
